lib/ble.toit

```diff
@@ -302,10 +447,13 @@ class Device:
   Initializes an advertiser for the local device.
 
   The returned advertiser is not started.
+
+  If the device has services defined, these are automatically added to the advertised data
```

Comment:
Terminate comment with .

Reply:
Actually, this is no longer true, as that lead to some rather convoluted code in the Advertiser class and the intention from the client is not always clear. So I removed the functionality, but not the comment.